jsa/features/people.feature

```diff
@@ -81,3 +81,173 @@ Feature: People
    Given I am logged in as a user with the "administrator" role
    When I go to "admin/manage/people"
    Then I should see 1 or more ".views-field-field-s-person-affiliation" elements
+
+  @content @deploy
+  Scenario: Faculty node on profiles page
+    Given I am on "people/faculty/profiles"
```

Comment:
Each of these specific layout Scenarios should check for CSS specific to the layout.